kep/624-disk-io-aware-scheduling/README.md

```diff
@@ -53,24 +53,81 @@ The mathematical relationship between the disk’s IO BW capacity and the runnin
 
 ## Proposal
 
-The disk IO aware scheduler plugin would implement the filter, score and reserve hook points of the scheduler framework. At the filter stage, it would obtain each node’s available disk IO capacity from the IO Driver at run time, update the available disk IO capacity of each node in its local cache, and filter out nodes which do not have enough IO capacity from the node candidate list. At the score stage, it would prioritize the node candidates based on a scoring policy, such as most allocated policy. At the reserve stage it would notify the node IO Driver of the new pod so that the IO Driver can start collecting the disk IO metrics of this new pod.
+The disk IO aware scheduler plugin would implement the filter, score and reserve hook points of the scheduler framework. At the filter stage, it would obtain each node’s available disk IO capacity from the API server, update the available disk IO capacity of each node in its local cache, and filter out nodes which do not have enough IO capacity from the node candidate list. At the score stage, it would prioritize the node candidates based on a scoring policy, such as most allocated policy. At the reserve stage it would notify the node IO Driver of the new pod via the API server so that the IO Driver can start collecting the disk IO metrics of this new pod.
```

Comment:
from ETCD through AI server? replace "update the available disk IO capacity of each node in its local cache" with "saves the info in its local cache, and filters out nodes...".  Does the IO driver watches some object ?